prow/plugins.yaml

```diff
@@ -13,6 +13,10 @@ triggers:
   - GoogleCloudPlatform/k8s-multicluster-ingress
   trusted_org: kubernetes
   join_org_url: "https://git.k8s.io/community/community-membership.md#member"
+ - repos:
+  - containerd/cri-containerd
+  trusted_org: containerd
+  join_org_url: "https://github.com/containerd/containerd/blob/master/MAINTAINERS"
```

Comment:
@cjwagner assume we can define multiple triggers here and this is valid

Reply:
If not, I could keep it to `kubernetes` org. :)
Do we have a policy for this?

Reply:
Changed to kubernetes org for now. I've pinged @cjwagner internally, but he hasn't replied yet. :)
Let's keep this safe and move fast, because we are blocked on this.

I'll send another PR to change this to containerd org once I make sure it's safe.